pkg/stdcopy/stdcopy.go

```diff
@@ -40,8 +35,11 @@ func (w *StdWriter) Write(buf []byte) (n int, err error) {
 	if w == nil || w.Writer == nil {
 		return 0, errors.New("Writer not instantiated")
 	}
-	binary.BigEndian.PutUint32(w.prefix[4:], uint32(len(buf)))
-	n1, err = w.Writer.Write(w.prefix[:])
+
+	header := [8]byte{0: w.prefix}
```

Comment:
yeah, there are a few hardcoded values that we might be able to replace by constants.